contentcuration/contentcuration/frontend/channelEdit/vuex/currentChannel/actions.js

```diff
@@ -1,29 +1,13 @@
+import Vue from 'vue';
 import client from 'shared/client';
-import logEvent from 'shared/analytics/tagManager';
 import applyChanges from 'shared/data/applyRemoteChanges';
 import { Channel } from 'shared/data/resources';
 
-let pageLoadEventFired = false;
-
 export function loadChannel(context, { staging = false } = {}) {
   return context
     .dispatch('channel/loadChannel', context.state.currentChannelId, { root: true })
     .then(channel => {
-      if (!pageLoadEventFired) {
```

Comment:
Just to check, do we still need to track pageLoadEventFired for anything else?

Reply:
Nope!